docs/docs/reference/erased-terms.md

```diff
@@ -193,8 +192,7 @@ if `def f(erased x: T): U` then `f: (erased T) => U`.
 
 
 6. Overloading
-
-Method with `erased` parameters will follow the normal overloading constraints after erasure.
+   Method with `erased` parameters will follow the normal overloading constraints after erasure.
```

Comment:
I would also put it on the previous line:
```
6. Overloading: Method with `erased` parameters will follow the normal overloading constraints after erasure.
```

Reply:
same here